Description: Change spelling of "setseid" to "setsid".
 Fixed upstream in r2334
Author: Stig Sandbeck Mathisen <ssm@debian.org>
Bug-Debian: http://bugs.debian.org/533467
--- a/node/munin-node.conf.in
+++ b/node/munin-node.conf.in
@@ -7,7 +7,7 @@
 pid_file @@STATEDIR@@/munin-node.pid
 
 background 1
-setseid 1
+setsid 1
 
 user root
 group root
--- a/node/munin-node.conf.pod
+++ b/node/munin-node.conf.pod
@@ -90,7 +90,7 @@
 	port 4949
 	pid_file /var/run/munin-node.pid
 	background 1
-	setseid 1
+	setsid 1
 
 	host *
 	user root
